lib/msf/core/modules/loader/base.rb

```diff
@@ -22,6 +22,7 @@ class Msf::Modules::Loader::Base
     Msf::MODULE_POST => 'post',
     Msf::MODULE_EVASION => 'evasion'
   }
+  TYPE_BY_DIRECTORY = DIRECTORY_BY_TYPE.invert
```

Comment:
what does this actually do? I tried reading the comment where you moved it from and it only confused me more

Reply:
`DIRECTORY_BY_TYPE` is a dictionary that maps the payload set type to the folder it can be found in.
So if you wanted to find out the directory for the `Msf::MODULE_PAYLOAD` modules, it's in the `payloads` folder.

`TYPE_BY_DIRECTORY` is the inverse of that, it'll convert a folder name into the module set type.

I don't think it's a great name, I tried renaming it to something better, but it's everywhere - and also copied into gems iirc